src/seedu/addressbook/AddressBook.java

```diff
@@ -1150,8 +1150,8 @@ private static String getUsageInfoForExitCommand() {
      * @param sign  Parameter sign to be removed
      * @return  string without the sign
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    private static String removePrefixSign(String person, String prefix) {
```

Comment:
are you sure the parameter should be named `person`? why should it be named `person` instead?